content/sensu-go/5.4/reference/events.md

```diff
@@ -50,267 +63,788 @@ Metric Format. Both the check results, and resulting (extracted) metrics are
+    {
+      "name": "sensu-go-sandbox.curl_timings.time_namelookup",
+      "tags": [],
+      "timestamp": 1552506033,
+      "value": 0.004
+    }
+  ]
+}
+{{< /highlight >}}
+
+### Check attributes
+
+Sensu events include a `check` scope containing information about how the event was created, including any attributes defined in the [check specification](../checks#check-specification), and information about the event and its history, including the attributes defined below.
+
+duration     |      |
+-------------|------
+description  | Command execution time in seconds
+required     | false
+type         | Float
+example      | {{< highlight shell >}}"duration": 1.903135228{{< /highlight >}}
+
+executed     |      |
+-------------|------
+description  | Time that the check request was executed
+required     | false
+type         | Integer
+example      | {{< highlight shell >}}"executed": 1522100915{{< /highlight >}}
+
+history      |      |
+-------------|------
+description  | Check status history for the last 21 check executions. See the [history attributes](#history-attributes).
+required     | false
+type         | Array
+example      | {{< highlight shell >}}
+"history": [
+  {
+    "executed": 1552505983,
+    "status": 0
+  },
+  {
+    "executed": 1552505993,
+    "status": 0
+  }
+]
+{{< /highlight >}}
+
+issued       |      |
+-------------|------
+description  | Time that the check request was issued in seconds since the Unix epoch
+required     | false
+type         | Integer
+example      | {{< highlight shell >}}"issued": 1552506033{{< /highlight >}}
+
+last_ok      |      |
+-------------|------
+description  | The last time that the check returned an OK `status` (`0`) in seconds since the Unix epoch
+required     | false
+type         | Integer
+example      | {{< highlight shell >}}"last_ok": 1552506033{{< /highlight >}}
+
+occurrences  |      |
+-------------|------
+description  | The number of times an event with the same status has occurred for the given entity and check
+required     | false
+type         | Integer
+example      | {{< highlight shell >}}"occurrences": 1{{< /highlight >}}
+
+occurrences_watermark | |
+-------------|------
+description  | The highest number of occurrences for the given entity and check at the current status
+required     | false
+type         | Integer
+example      | {{< highlight shell >}}"occurrences_watermark": 1{{< /highlight >}}\
+
+output       |      |
+-------------|------
+description  | The output from the execution of the check command
+required     | false
+type         | String
+example      | {{< highlight shell >}}
+"output": "sensu-go-sandbox.curl_timings.time_total 0.005"
+{{< /highlight >}}
+
+state         |      |
+-------------|------
+description  | The state of the check: `passing`, `failing`, or `flapping` (a rapid change in check result status)
+required     | false
+type         | String
+example      | {{< highlight shell >}}"state": "passing"{{< /highlight >}}
```

Comment:
Is there a relationship between `state` and `status`?

Reply:
It depends; we [first check](https://github.com/sensu/sensu-go/blob/f5b8db4d5d75c9ed9960ff6d5ed5bd905a3f9812/backend/eventd/flapping.go#L31) if the check is flapping using the check `TotalStateChange`, `LowFlapThreshold` & `HighFlapThreshold`. If not, then [we use the `status`](https://github.com/sensu/sensu-go/blob/f5b8db4d5d75c9ed9960ff6d5ed5bd905a3f9812/backend/eventd/flapping.go#L33) to determine whether is passing or failing.

Reply:
Excellent, I added a note in https://github.com/sensu/sensu-docs/pull/1327/commits/a44a2415e6114bb13804c5fc8fda5a4900e4c394

Reply:
👌